src/server/adminInviteUser.ts

```diff
@@ -42,24 +42,27 @@ export async function adminInviteUser(
         };
     }
 
-    if (data) {
-        const { error: createRoleError } = await supabase.from("profiles").insert({
-            primary_key: data.user.id,
-            role: userDetails.role,
-            first_name: userDetails.firstName,
-            last_name: userDetails.lastName,
-            telephone_number: userDetails.telephoneNumber,
-        });
-        if (createRoleError) {
-            return {
-                data: null,
-                error: { "Failed to create user profile": createRoleError.message },
-            };
-        }
-        void logInfoReturnLogId(
-            `Created a profile for ${userDetails.role} user: ${userDetails.email}`
-        );
+    if (!data.user) {
+        return {
+            data: null,
+            error: { "Failed to create user": "Email may be in use already." },
+        };
```

Comment:
Not ready for team review, I need to check this behaviour on trying to create a user with an email address that already is in use

Reply:
I have removed this as it was not working as intended. Created a new ticket for the error handling on creating a duplicate user, as this is a little trickier than first thought - [VFB-189](https://softwiretech.atlassian.net/browse/VFB-189)